patchwork-key-bindings/src/main/java/net/patchworkmc/mixin/keybindings/MixinKeybinding.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Minecraft Forge, Patchwork Project
+ * Copyright (c) 2016-2020, 2019-2020
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation version 2.1
+ * of the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package net.patchworkmc.mixin.keybindings;
+
+import net.minecraftforge.client.extensions.IForgeKeybinding;
+import net.minecraftforge.client.settings.IKeyConflictContext;
+import net.minecraftforge.client.settings.KeyConflictContext;
+import net.minecraftforge.client.settings.KeyModifier;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+
+import net.minecraft.client.options.KeyBinding;
+import net.minecraft.client.util.InputUtil;
+
+import net.patchworkmc.impl.keybindings.ForgeKeyBindingConstruct;
+
+@Mixin(KeyBinding.class)
+public abstract class MixinKeybinding implements Comparable<KeyBinding>, IForgeKeybinding, ForgeKeyBindingConstruct {
```

Comment:
Is there any reason this doesn't match the casing of KeyBinding?

Reply:
Also see `IForgeKeybinding`, can't fix that, so: either pick `Keybinding` or `KeyBinding`

Reply:
I'd say MixinKeyBinding here since the general rule for mixins is `MixinClassName`.
```suggestion
public abstract class MixinKeyBinding implements Comparable<KeyBinding>, IForgeKeybinding, ForgeKeyBindingConstruct {
```